README.md

```diff
@@ -107,6 +107,16 @@ Tests (with embedded Cassandra)
 ./sbt test
 ```
 
+To check if the code follows the project style, use the linters
+```
+make lint-scala
+```
+
+To fix formatting issues, use the formatters
+```
+make format-scala
+```
+
```

Comment:
I suggest we 
 - add details on how to configure IDE plugin with the same set of formater/liner ruler here as well
 - make sure that after those steps re-formatting between CLI and IDE does not happen